classes/db/DBConnection.inc.php

```diff
@@ -56,6 +57,16 @@ function __construct() {
 		}
 	}
 
+	/**
+	 * Log message handler for ADODB.
+	 * @param $msg string
+	 * @param $newline boolean
+	 * @see ADOConnection::outp
+	 */
+	static function logAdodbMessage($msg, $newline=true) {
+		error_log(Registry::get('system.debug.startTime') . ': ' . PKPString::html2text($msg));
+	}
+
```

Comment:
@asmecher It works perfectly fine. Could I suggest 2 things:

- Perhaps adding some prefix for those logs, in order to distinguish from other PHP warnings/errors - Something like: `error_log('PKP-Database-Logger ' . Registry::get('system.debug.startTime') . ': ' . PKPString::html2text($msg));` 
- Perhaps we could add some data regarding the caller of the sql. Something like `
$backtrace = debug_backtrace(2);
error_log(print_r($backtrace, true) . '-> ' . Registry::get('system.debug.startTime') . ': ' . PKPString::html2text($msg));` (even though that could result to a rather extensive log. Maybe we could traverse on the `$backtrace` in order to display less, and more useful info)

Reply:
Thanks, @defstat! I agree with the first; let's come back to the second if there's trouble locating the SQL. I find grepping for a piece of the SQL works pretty well.